run-spread

```diff
@@ -1,25 +1,25 @@
 #!/bin/bash
 
-# check dependencies
-if ! snap list snapcraft &>/dev/null; then
-    echo "snapcraft is not installed"
-    exit 1
-fi
+set -e
 
-# Clean the snaps created in previous runs
-rm -rf built-snap test-build
-touch test-build
-mkdir built-snap
+need_rebuild=0
 
-rm -f snapd_1337.*.snap 
-rm -f snapd_1337.*.snap.keep
+if [ -d built-snap ]; then
```

Comment:
Not sure this is sufficient for not requiring rebuild? 
What about timestamp compared to latest commit / git tracked file change?

Reply:
Default changed to rebuild.